index.bs

```diff
@@ -6388,6 +6414,10 @@ for="value-with-size">value</dfn> and <dfn for="value-with-size">size</dfn>.
  performs the following steps:
 
  1. Assert: |container| has \[[queue]] and \[[queueTotalSize]] internal slots.
+ 1. If |container|.\[[isTransferring]] is true, perform the following steps until |container|.\[[queue]]
```

Comment:
Same problem as above.

Maybe add a `ReadableStreamDefaultControllerResetQueue` abstract op which "wraps" `ResetQueue`?

Reply:
We will probably want to reuse the same reset queue algorithm for writable stream default controller.
That said, ResetQueue is also used for Byob. Given there will be no isTransferring slot, this should have no impact. But I wonder whether this is in line with this spec style.

Reply:
I updated to `if |container| has a \[[isTransferring]] slot whose value is true` to make that clearer